docs/developing.md

```diff
@@ -13,6 +13,8 @@ You must have the following installed in the development environment:
 
 ## Install packages
 
+### Python
```

Comment:
Is there any other language sections? If not perhaps we rename the `##` section to "Install Python packages"